api/backend.go

```diff
@@ -758,3 +758,20 @@ func (b *StatusBackend) UpdateMailservers(enodes []string) error {
 	}
 	return st.UpdateMailservers(nodes)
 }
+
+// SignHash exposes vanilla ECDSA signing for signing a message for Swarm
+func (b *StatusBackend) SignHash(hexEncodedHash string) (string, error) {
+	hash, err := hexutil.Decode(hexEncodedHash)
```

Comment:
is it necessary?

Reply:
it transforms the string like `0xABABABABABABABABAB....` to an actuall array of bytes `[]byte{ 0xAB, 0xAB, 0xAB}` that we are are using for hashing.

the function itself receives and returns hex-strings: `0xABABABABAB` -> `0x12121212122121`

Reply:
ok perfect then thanks for explaining